lang/src/org/partiql/lang/eval/ExprValueFactory.kt

```diff
@@ -45,10 +45,10 @@ import java.math.BigInteger
 import java.time.LocalDate
 import kotlin.collections.asSequence
 
-internal const val MISSING_ANNOTATION = "\$partiql_missing"
-internal const val BAG_ANNOTATION = "\$partiql_bag"
-internal const val DATE_ANNOTATION = "\$partiql_date"
-internal const val TIME_ANNOTATION = "\$partiql_time"
+const val MISSING_ANNOTATION = "\$missing"
```

Comment:
Does this need to be public? It was previously internal?

Reply:
I think it should be public

Reply:
Could you please elaborate why? I would think the annotations are either all public, or all internal.

Reply:
They are all public

Reply:
PartiQL data should be able to be represented by `IonValue`. The annotation itself is considered the part of this representation, otherwise, we won't be able to differentiate 2 types. e.g. the only way to differentiate PartiQL BAG & LIST types in representation of `IonValue` is to check the annotation of the `IonList`. If it has `$bag` there, that means it is a BAG, otherwise, it is a LIST. This representation should be publicly visible to users.

Reply:
I agree that the representation should probably be public. I was just curious as to why these variables were internal previously and in this PR we had to make them public. I had thought the intent of this PR was to just remove the `partiql_` prefix from the Ion representation.

For Robert's comment, think there might be some confusion over the lines I selected for the PR comment. All the annotations were made public in this PR, but my comment only highlighted the first annotation.